backend/ops_api/.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 select = B,B9,BLK,C,E,F,S,W
-ignore = E203,E501,W503
+ignore = E203,E501,W503,BLK100
```

Comment:
Not sure we want to ignore Black formatting changes

Reply:
fixed.